services/mcp/Services/GamerUncleMcpServer.cs

```diff
@@ -55,6 +55,8 @@ public async Task<object> ProcessJsonRpcAsync(string requestBody)
                     "initialize" => await HandleInitialize(root, idValue),
                     "tools/list" => await HandleToolsList(root, idValue),
                     "tools/call" => await HandleToolCall(root, idValue),
+                    "prompts/list" => await HandlePromptsList(root, idValue),
+                    "notifications/initialized" => await HandleNotificationsInitialized(root, idValue),
```

Comment:
Do we necessarily need these additional endpoints? It appears like they are stubbed to return empty objects.